### DISCUSSION TITLE
What is the stiffness equivalent for the Generic6DOFJoint for jolt

### DISCUSSION THREAD

**Discussion by just-another-player** [godot-jolt/godot-jolt#819] [Questions]
I'm following a tutorial for ragdolls in godot which I haven't done, it was made in godot 3 but I'm doing godot 4 so I'm using jolt however they add stiffness not frequency which seems to be something different entirely, can someone tell me what the stiffness for jolt is for this joint?

**Comment by mihe** [comment]
You can still use `Generic6DOFJoint` and its stiffness property, which still uses Jolt under the hood. The `JoltGeneric6DOFJoint` node is just there to provide things like soft limits and a few other things that we weren't able to support for the regular `Generic6DOFJoint`.

I wouldn't expect to be able to just copy values from Bullet in Godot 3 to Jolt though. They are quite different in their implementation.
